src/pydvl/influence/model_wrappers/torch_wrappers.py

```diff
@@ -106,8 +108,13 @@ def fit(
                 if scheduler:
                     scheduler.step()
             pred_val = self.forward(x_val)
-            val_loss.append(loss(torch.squeeze(pred_val), torch.squeeze(y_val)).item())
-            train_loss.append(np.mean(batch_loss))
+            epoch_val_loss = loss(torch.squeeze(pred_val), torch.squeeze(y_val)).item()
+            mean_epoch_train_loss = np.mean(batch_loss)
```

Comment:
After a certain a point, it might make sense to just rely on pytorch-lightning for these things.

Reply:
Wrote [issue about this.](https://github.com/appliedAI-Initiative/pyDVL/issues/222).